ui/pages/routes/routes.component.js

```diff
@@ -280,9 +280,8 @@ export default class Routes extends Component {
             exact
           />
           <Authenticated
-            path={REVEAL_SEED_ROUTE}
+            path={`${REVEAL_SEED_ROUTE}/:keyringId?`}
```

Comment:
```suggestion
            // `:keyringId` is optional here, if not provided, this will fallback
            // to the main seed phrase.
            path={`${REVEAL_SEED_ROUTE}/:keyringId?`}
```